src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const malePerson = people.filter(person => {
+    return century
+      ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+      : person.sex === 'm';
+  });
+
+  const menAges = malePerson.map(person => person.died - person.born);
+  const sumOfAges = menAges.reduce((sum, age) => sum + age, 0);
+  const averageAge = sumOfAges / menAges.length;
```

Comment:
you also have this in `calculateWomenAverageAge`, move this into separate function